docs/topics_5/web-openshift-troubleshoot.adoc

```diff
@@ -135,7 +135,7 @@ This error can be caused by a number of sources, with the following being the mo
 
 Attempt the deployment again, and view the logs and events of the non deployment Pods while they are being created. These messages will provide context to the underlying errors resulting in the deployment Pod timeouts.
 
-* To address the first issue reported, where the OpenShift instance is out of resources, follow the instructions in link:{OpenShiftDocsURL}/nodes/clusters/nodes-cluster-resource-levels.html[Analyzing Cluster Capacity] to determine the cluster capacity. Once the capacity has increased, or there are fewer jobs executing, attempt the deployment once again.
+* To address the first issue reported, where the OpenShift instance is out of resources, follow the instructions in link:{OpenShiftDocsURL}/nodes/clusters/nodes-cluster-resource-levels.html[Analyzing Cluster Capacity] in the OpenShift documentation to determine the cluster capacity. Once the capacity has increased, or there are fewer jobs executing, attempt the deployment once again.
 
 * To address the second issue reported, where the images are unable to be pulled from the registry, link:{OpenShiftDocsURL}/registry/accessing-the-registry.html[access the registry] to ensure the images are present. This link also includes instructions on examining the logs for the Docker registry, and can be used to troubleshoot the issue further.
```

Comment:
Same thing. "To address the second issue" is bad writing. Change to, "If the images cannot be pulled from the registry, link:[access the registry] to verify that the images are present and link:{URL}/registry/accessing-the-registry.html#registry-viewing-logs_accessing-the-registry[check the logs].

That last sentence is not helpful. Remove it. Add the useful bit about checking logs to the first sentence with a **link**.